server.ts

```diff
@@ -42,9 +42,15 @@ const CONFIG = {
   'activity_next_poll_request_ms': parseInt(process.env['CCD_ACTIVITY_NEXT_POLL_REQUEST_MS'], 10) || 5000,
   'activity_retry': parseInt(process.env['CCD_ACTIVITY_RETRY'], 10) || 5,
   'activity_batch_collection_delay_ms': parseInt(process.env['CCD_ACTIVITY_BATCH_COLLECTION_DELAY_MS'], 10) || 1,
-  'activity_max_request_per_batch': parseInt(process.env['CCD_ACTIVITY_MAX_REQUEST_PER_BATCH'], 10) || 25
+  'activity_max_request_per_batch': parseInt(process.env['CCD_ACTIVITY_MAX_REQUEST_PER_BATCH'], 10) || 25,
+  'web_pages_instrumentation_key': process.env['WEB_PAGES_APPINSIGHTS_INSTRUMENTATION_KEY'] || '',
```

Comment:
Please don't use another environment variable name, we already have 3,
https://github.com/hmcts/cnp-module-webapp/blob/master/main.tf#L64-L69

Reply:
The other 3 are quite similar to each other. With this one, I wanted to have an express separation between the key for the AI resource of the BE service and the key for that of the FE pages. These keys may be different indeed (as suggested by MS as well).